packages/lib/src/classes/EventHandler.ts

```diff
@@ -0,0 +1,64 @@
+import { ClientEvents } from "discord.js"
+import { BetterClient, LogLevel } from "../index.js"
+
+export default class EventHandler {
```

Comment:
discussion was had about having the ability to bind custom events, like kiai's addXp event, through here to be emitted through the client, not at all essential or really of high priority but figured id leave the comment here nonetheless.

Reply:
We could use a similar approach to sapphire where an event would have a `emitter` option that would default to `ws` for client events.